modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/index.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import 'ckeditor5/ckeditor5.css';
+
+// @ts-ignore
+
+export {default as BalloonEditor} from './ckeditor4/BalloonEditor';
+
+// @ts-ignore
+
+export {default as ClassicEditor} from './ckeditor4/ClassicEditor';
+
+// @ts-ignore
+
+export {Editor} from './ckeditor4/Editor';
+
+// @ts-ignore
+
+export {InlineEditor} from './ckeditor4/InlineEditor';
+
+export {default as CKEditor5ClassicEditor} from './ckeditor5/ClassicEditor';
```

Comment:
I will pull down and take a look

Reply:
The only thing that comes to mind is that `{default as x}`  is not the same as plain `default x` -> https://dev.to/zacharylee/the-differences-between-export-default-xx-and-export-xx-as-default-13a7

But it may be unrelated to your error, though...